WordPress/src/main/res/layout/reader_popup_menu_spacer_item.xml

```diff
@@ -0,0 +1,11 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/reader_popup_menu_item_spacer_container"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="start|center_vertical"
+    android:orientation="horizontal">
+    <android.widget.Space xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Minor (🔍): This namespace here is redundant. Consider deleting it as per the IDE quick fix.